sonar-plugin-api/src/main/java/org/sonar/api/config/Configuration.java

```diff
@@ -19,13 +19,18 @@
  */
 package org.sonar.api.config;
 
+import java.util.Collections;
+import java.util.HashMap;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.HashMap'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)